kong/clustering/utils.lua

```diff
@@ -90,7 +90,7 @@ function _M.connect_cp(dp, endpoint, protocols)
       wss_proxy_authorization = proxy_opts.proxy_authorization,
     }
 
-    ngx_log(ngx_DEBUG, _log_prefix,
+    ngx_log(ngx_INFO, _log_prefix,
```

Comment:
same here, only log when we receive different hash.

Reply:
i'm not familiar with the gateway, so please correct me if i'm wrong. this function is used to open a new connection to the controplane, which is unrelated to receiving a new hash?